bots/twitterbot.go

```diff
@@ -100,6 +100,8 @@ func NewTwitterBot(path string) (Bot, error) {
 	bot.coniksAddress = conf.CONIKSAddress
 	bot.handle = conf.Handle
 
+	bot.deleteOldDMs()
```

Comment:
Is `deleteOldDMs` a safeguard in case `deleteRequestDMs` doesn't manage to delete all DMs? I'm wondering how often this function will actually be necessary given that `deleteRequestDMs` already deletes response DMs.